tests/unit/cocoon/test_utilities.py

```diff
@@ -1645,41 +1665,38 @@ def test_identify_cash_items_failed(
                     "implicit": "internal_currency",
                 },
                 True,
-                [],
+                ["inst5"],
             ),
         ]
     )
-    def test_identify_cash_items(self, _, cash_flag, remove_cash_items, ground_truth):
+    def test_identify_cash_items_remove_true(self, _, cash_flag, remove_cash_items, ground_truth):
```

Comment:
as above for naming and removing `remove_cash_items` parameter

Reply:
`test_identify_cash_items_without_remove()` as above